### PR TITLE
DA-4646: jira connector updates

### PR DESCRIPTION
**Pull request by ajinkyan83**
Data pushed here:

https://s3.console.aws.amazon.com/s3/buckets/datalake-input-dev?prefix=insights/jira/issues/2022/02/08/&region=us-east-2

Signed-off-by: Ajinkya Nahar <ajinkyan@proximabiz.com>

### CODE REVIEW

File: cmd/jira/jira.go [hunk 1]
```diff
@@ -289,7 +325,7 @@ func (j *DSJira) EnrichComments(ctx *shared.Ctx, comments []interface{}, item ma
 	for _, comment := range comments {
 		richComment := make(map[string]interface{})
 		for _, field := range shared.RawFields {
-			v, _ := item[field]
+			v := item[field]
```
Comment (lukaszgryglicki) [inline cmd/jira/jira.go:328]: why removing `, _` part? That can cause panics.

Comment (ajinkyan83) [reply cmd/jira/jira.go:328]: Static compiler checks, gave me this suggestion.

Comment (ajinkyan83) [reply cmd/jira/jira.go:725]: okay, will revert it.

**Comment by ajinkyan83** [comment]
Command to test
```
./jira --jira-url=${JIRA_URL} --jira-es-url="${ES_URL}"  --jira-debug=2 --jira-project="avalon" --jira-project-filter=true --jira-st=true --jira-dry-run=true
```

File: cmd/jira/jira.go (hunk 1 above)
Comment (ajinkyan83) [reply cmd/jira/jira.go]: done!

**Comment by ajinkyan83** [comment]
@lukaszgryglicki @fayazg PR is ready for review.
I am able to successfully push `hyperledger/avalon` project issues to s3 
URL:
https://s3.console.aws.amazon.com/s3/buckets/datalake-input-dev?prefix=insights/jira/issues/2022/02/08/&region=us-east-2

**Review by lukaszgryglicki — APPROVED** [review]
/lgtm

**Pull request merged**